packages/shared/src/logic/deeplinks.ts

```diff
@@ -23,6 +19,7 @@ export async function getReferenceFromDeeplink({
   });
 
   if (linkMeta && typeof linkMeta === 'object') {
+    // TODO: handle personal invite links
```

Comment:
do you need to do this before deploy?

Reply:
Na, we can address later. Handling these pastes isn't critical, it'll bail when it sees the missing `invitedGroupId` anyhow